src/Symfony/Component/Console/Command/Command.php

```diff
@@ -79,36 +90,11 @@ public static function getDefaultDescription(): ?string
      */
     public function __construct(?string $name = null)
```

Comment:
@ondrejmirtes Is there a way to allow not calling the parent constructor with PHPStan strict rules?
https://phpstan.org/r/77990151-3307-4040-a6bf-534577711e83

Reply:
It seems we can't: [`RequireParentConstructCallRule'](https://github.com/phpstan/phpstan-strict-rules/blob/2.0.x/src/Rules/Classes/RequireParentConstructCallRule.php#L63)

And this was discouraged: https://github.com/phpstan/phpstan/issues/228#issuecomment-292642260